modules/app.xql

```diff
@@ -59,6 +61,12 @@ declare function app:scheduler-enabled($node as node(), $model as map(*)) {
         $node
 };
 
+declare 
+    %templates:warp
+function app:java-version ($version as xs:string?, $model as map(*)) as node()? {
```

Comment:
I meant to ask why a parameter was being passed in at all. The function doesn't need version to be passed in; it generates this result by itself.